book/src/configuration.md

```diff
@@ -116,7 +118,7 @@ The following statusline elements can be configured:
 | `position` | The cursor position |
 | `position-percentage` | The cursor position as a percentage of the total number of lines |
 | `separator` | The string defined in `editor.statusline.separator` (defaults to `"│"`) |
-| `spacer` | Inserts a space between elements (multiple/contiguous spacers may be specified) |
+| `spacer` | An empty string, used to join multiple spacers |
```

Comment:
```suggestion
| `spacer` | A string used as space to separate all statusline elements, defined in `editor.statusline.spacer` (defaults to `" "`) |
```

Reply:
The option has been removed, so this is no longer an issue.

Reply:
I think I prefer the original docs here. The new docs aren't quite accurate and since the behavior hasn't been changed I see no reason to change the docs either

Reply:
Yeah even if this is what it technically does code wise, the docs are user facing and should explain what is actually important to user which is it adds another space

Reply:
Ended up leaving the original description.